packages/ses/src/module-load.js

```diff
@@ -514,9 +514,10 @@ const asyncJobQueue = (errors = []) => {
   /**
    * Enqueues a job that starts immediately but won't be awaited until drainQueue is called.
    *
-   * @template {any[]} T
-   * @param {(...args: T)=>Promise<*>} func
-   * @param {T} args
+   * @template {(...args: any[]) => Promise<any>} F
```

Comment:
```suggestion
   * @template {(...args: any[]) => Promise<void>} F
```

Reply:
my reasoning is that we do not want to encourage the use of a return value in any of these "jobs" because it will be ignored